packages/app/src/app/pages/common/Modals/PreferencesModal/Appearance/index.tsx

```diff
@@ -96,8 +95,7 @@ export const Appearance: React.FC = () => {
             <PaddedPreference
               title="Line Height"
               type="number"
-              placeholder="1.15"
```

Comment:
We need these ones :(

The typing may need to be changed

Reply:
Because we are using `type="number"` here, so it will be a `<PreferenceNumber />`([component](https://github.com/codesandbox/codesandbox-client/blob/master/packages/common/src/components/Preference/PreferenceNumber.tsx)) under the hood.
And we do not pass `placeholder` to `<StyledInput />` (which is a `input`) [currently](https://github.com/codesandbox/codesandbox-client/blob/master/packages/common/src/components/Preference/PreferenceNumber.tsx#L24). As you said that we actually need it, I can fix that in this PR if you do not mind.
Otherwise I can make another PR to add `placeholder` in `<PreferenceNumber />` if that's the preferable way to do it.

Reply:
@SaraVieira Updated with `placeholder` for `<PreferenceNumber />`.